playbooks/installer/ospd/post_install/scale_swift.yml

```diff
@@ -0,0 +1,20 @@
+---
+- name: Scaling swift-storage nodes down
+  hosts: undercloud
+  become: yes
+  become_user: "{{ installer.user.name }}"
+  gather_facts: no
+  roles:
+      - {role: "installer/ospd/overcloud/scale/", service: 'swift-storage', amount: 1}
+
+- name: Scaling swift-storage nodes up
+  hosts: undercloud
+  become: yes
+  become_user: "{{ installer.user.name }}"
+  gather_facts: no
+  pre_tasks:
+      - name: Wait for min_part_hours (the swift timeout between partition reassignment) timeout
```

Comment:
60 min pause is problematic.
I would like to see a proper waiter that makes it clear what we are waiting for (partition creation? file creation? boot?) and not a hard coded pause. Please use `wait_for` module or `retry`